quickwit/quickwit-search/src/search_client_pool.rs

```diff
@@ -167,41 +176,15 @@ impl SearchClientPool {
     /// When a client pool is created, the thread that monitors cluster members
     /// will be started at the same time.
     pub async fn create_and_keep_updated(
-        current_members: &[ClusterMember],
```

Comment:
Passing `current_members` and a member `WatchStream` is, IMHO, a wrong usage of a watch stream.
The watch stream always start with the last seen values. And we always send at least the "self" member, so we always have something. At least self.